python-lib/dku_kube/gpu_driver.py

```diff
@@ -29,8 +29,9 @@ def add_gpu_driver_if_needed(cluster_id, kube_config_path, connection_info, tain
     # Retrieve the tolerations on the daemonset currently deployed to the cluster.
     if has_gpu_driver(kube_config_path):
         cmd = ['kubectl', 'get', 'daemonset', 'nvidia-device-plugin-daemonset', '-n', 'kube-system', '-o', 'jsonpath="{.spec.template.spec.tolerations}"']
-        tolerations_json, err = run_with_timeout(cmd, env=env, timeout=5)
-        if _is_none_or_blank(tolerations_json):
+        cmd_result, err = run_with_timeout(cmd, env=env, timeout=5)
+        tolerations_json = cmd_result[1:-1]
```

Comment:
received response is actually `"\"{...}\""` and the JSON parser complains about the additional characters so escaping them